cc/pass_alive_test.cc

```diff
@@ -0,0 +1,732 @@
+       R"(.........
+          .........
+          ...XX....
+          ..X..X...
+          ..X..X...
+          ...XX....
+          .........
+          .........
+          .........)"},
+
+      {// board state
+       R"(.........
+          .........
+          .XXXX....
+          .X..X....
+          .X..XXX..
+          .XXXX.X..
+          ....XXX..
+          .........
+          .........)",
+       // expected result
+       R"(.........
+          .........
+          .XXXX....
+          .XXXX....
+          .XXXXXX..
+          .XXXXXX..
+          ....XXX..
+          .........
+          .........)"},
+
+      {// board state
+       R"(.........
+          .XXXXX...
+          .X...X...
+          .X...X...
+          .X...XXX.
+          .XXXXX.X.
+          .....XXX.
+          .........
+          .........)",
+       // expected result
+       R"(.........
+          .XXXXX...
+          .X...X...
+          .X...X...
+          .X...XXX.
+          .XXXXX.X.
+          .....XXX.
+          .........
+          .........)"},
+
+      {// board state
+       R"(.........
+          .........
+          ....X....
+          .........
+          .........
+          .........
+          .........
+          .........
+          .........)",
+       // expected result
+       R"(.........
+          .........
+          ....X....
+          .........
+          .........
+          .........
+          .........
+          .........
+          .........)"},
+
+      {// board state
+       R"(.........
+          .........
+          .........
+          .........
+          .........
+          .........
+          .........
+          ....O....
+          .........)",
+       // expected result
+       R"(.........
+          .........
+          .........
+          .........
+          .........
+          .........
+          .........
+          ....O....
+          .........)"},
+
+      {// board state
+       R"(O........
+           ....OOOO.
+           .OOOOXXO.
+           .OXXX.XO.
+           .OX..XXO.
+           .OXXXOOO.
+           .OOOOO...
+           .........
+           .........)",
+       // expected result
+       R"(O........
+          ....OOOO.
+          .OOOOXXO.
+          .OXXX.XO.
+          .OX..XXO.
+          .OXXXOOO.
+          .OOOOO...
+          .........
+          .........)"},
+
+      {// Seki
+       // board state
+       R"(.OX.XO...
+          OOOXXO...
+          .OXXOO...
+          OXXXO....
+          XX.XO....
+          OOXXO....
+          .OOOO....
+          .........
+          .........)",
+       // expected result
+       R"(.OX.XO...
+          OOOXXO...
+          .OXXOO...
+          OXXXO....
+          XX.XO....
+          OOXXO....
+          .OOOO....
+          .........
+          .........)"},
+  };
+
+  RunTests(tests);
+}
+
+TEST_F(PassAliveTest, 19x19) {
+  if (kN != 19) {
+    return;
+  }
+
+  TestCase tests[] = {
+      {// board state
```

Comment:
since the parser ignores whitespace, can you add a space in between every character of this test case, to get a square aspect ratio? :D

Reply:
*sigh*